src/components/api-reference/schemas/measurements.js

```diff
@@ -186,8 +189,8 @@ class Measurements extends React.PureComponent {
         </div>
         <DataModelTable fields={PROPORTION_FIELDS} />
 
-        <h1 className='ds-h1' id='continuous-variable-measurements'>Continuous Variable Measurements</h1>
-        <p className='ds-text--lead'>Continuous Variable Measurements are applicable to quality measures.</p>
+        <h1 className='ds-h1' id='non-proportion-measurements'>Non-Proportion Measurements</h1>
+        <p className='ds-text--lead'>Non-Proportion Measurements are applicable to quality measures. They are exclusively authored by QCDRs and are used to attest to measures that are otherwise categorized as 'ratio', 'continuous variable', or a combination of 'proportion' and the former. This means that whether a measure is considered non-proportional does not depend solely on a categorized of false in the proportional field of QCDR documentation; a measurement that is both proportional and ratio, or both proportional and continuous, is considered a non-proportional measurement. Non-proportion measurements are unconstrained, so while the fields are 'numerator' and 'denominator' there is no validation that the numerator must be less than or equal to the denominator or that the denominator is greater than 0, as is the case for proportion measurements.</p>
```

Comment:
How about change this (uh, I might have suggested this language in another PR) part from:
`This means that whether a measure is considered non-proportional does not depend solely on a categorized of false in the proportional field of QCDR documentation`

to:
`Note this means that having a false value in the proportion field of QCDR documentation is sufficient to determine that measure as non-proportional, but having a true value for proportion is insufficient to determine that measure as proportional`

Reply:
Cool yeah I had edited it slightly to reference the QCDR documentation - I like your suggestion!